webpack/assets/javascripts/foreman_navigation.js

```diff
@@ -1,6 +1,18 @@
 import $ from 'jquery';
 import store from './react_app/redux';
 import * as LayoutActions from './react_app/components/Layout/LayoutActions';
+import { deprecate } from './react_app/common/DeprecationService';
+
+window.Turbolinks = {
```

Comment:
Sorry for noticing it just now but moving it to the `foreman_navigation.js` make me wonder.
Should we add something like?
```js
export const visit = url => {
 window.location.href = url;
};

// from legacy js
window.tfm.nav.visit('some-url');

// from webpack
import { visit } from '../foreman_navigation';

visit('some-url');
```

It might help in a future refactoring.

Reply:
Oh, right, thats nice! I do not really like it comming after I did it everywhere, but hey good idea, let's do it!

Reply:
Done that, it wasn't that bad :) and looks much better :wink: